ui/ui-components/__tests__/components/record/SampleRecordCard.tsx

```diff
@@ -24,18 +25,20 @@ describe("SampleRecordCard", () => {
     let card: ReturnType<typeof render>;
     await act(async () => {
       card = render(
-        <ApiContext.Provider
-          value={{
-            client: {
-              request: async () => ({
-                records: [],
-                total: 0,
-              }),
-            } as any,
-          }}
-        >
-          <SampleRecordCard {...cardProps} />
-        </ApiContext.Provider>
+        <GrouparooModelContext.Provider value={{ model: { id: modelId } }}>
+          <ApiContext.Provider
```

Comment:
At some point we should probably add a helper to simulate what _app provides, doesn't have to be done here. Maybe add a chore to our backlog?

Reply:
Could utilize `renderNestedContextProviders`?

Reply:
https://www.pivotaltracker.com/story/show/181234864